backend/application/syson-application-configuration/src/main/java/org/eclipse/syson/application/configuration/SysMLv2PropertiesConfigurer.java

```diff
@@ -321,6 +324,39 @@ private GroupDescription createExtraMembershipPropertiesGroup() {
         return group;
     }
 
+    private GroupDescription createExtraAcceptActionUsagePropertiesGroup() {
```

Comment:
The extra properties group `Payload` and `Receiver` are display in the Details View even if the `Accept Action` doesn't have a payload or a receiver.
In these cases, using the reference widgets associated to set values lead to errors.